### ISSUE TITLE
parameters explanation

### ISSUE THREAD

**Issue by mytk2012** [idstcv/DR_loss#8]
Can you explain what  does logits mean or what does it consist of?I don't use maskrcnn-benchmark，but i wanna transfer it.

**Comment by qian-qi** [comment]
Logits are the inputs to the classification layer.
